frontend/src/scenes/session-recordings/replayLandingPageLogic.ts

```diff
@@ -0,0 +1,39 @@
+import { connect, kea, path, reducers, selectors } from 'kea'
+import { FEATURE_FLAGS } from 'lib/constants'
+import { featureFlagLogic } from 'lib/logic/featureFlagLogic'
+import { sessionReplaySceneLogic } from 'scenes/session-recordings/sessionReplaySceneLogic'
+
+import { ReplayTabs } from '~/types'
+
+import type { replayLandingPageLogicType } from './replayLandingPageLogicType'
+
+/**
+ * We want to remember the last replay tab you were on and present that as your current "home" tab
+ * Alongside that we want to be able to control which tab we prefer if you've not visited the replay page before
+ */
+export const replayLandingPageLogic = kea<replayLandingPageLogicType>([
+    path(['scenes', 'session-recordings', 'landingPageLogic']),
+    connect({ values: [featureFlagLogic, ['featureFlags']], actions: [sessionReplaySceneLogic, ['setTab']] }),
+    reducers({
+        chosenTab: [
+            null as ReplayTabs | null,
+            { persist: true },
```

Comment:
there's no key here so this choice is used for every project this user is a member of, but that feels like the right behaviour... once you choose a tab, we stick with it